### PR TITLE
[team_hi_0]: helloworld pr for users

### PR DESCRIPTION
**Pull request by Adrian-cc**
# What?
brief: 第一次pr
author: Adrian-cc

# What changes are included in this PR?
包含了一个团队目录、个人目录、个人学习内容。

# Are these changes tested?
测试代码见hello_world.cpp，里面编写了一个helloworld程序

3Q{(E6OB(UBOLL5MY4 G1O6_tmb

### CODE REVIEW

File: stl/team_hi/adrian_hi/src/hello_world.cpp [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#include <iostream>
+
+int main(){
+	std::cout<<"hello world"<<std::endl;
```
Comment (Light-City) [inline stl/team_hi/adrian_hi/src/hello_world.cpp:4]: indent.

**Comment by Light-City** [comment]
remove bin file.

File: stl/team_hi/adrian_hi/src/hello_world.cpp (hunk 1 above)
Comment (Adrian-cc) [reply stl/team_hi/adrian_hi/src/hello_world.cpp]: Thanks, I have modified the indentation as well as removed the bin file

**Comment by Adrian-cc** [comment]
Thanks, I have modified the indentation as well as removed the bin file

File: stl/team_hi/adrian_hi/src/hello_world.cpp [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#include <iostream>
+
+int main(){
```
Comment (Light-City) [inline stl/team_hi/adrian_hi/src/hello_world.cpp:3]: main() {

blank

File: stl/team_hi/adrian_hi/src/hello_world.cpp [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+    std::cout<<"hello world"<<std::endl;
```
Comment (Light-City) [inline stl/team_hi/adrian_hi/src/hello_world.cpp:4]: 2 indent

**Comment by Light-City** [comment]
LGTM

**Pull request closed without merging**